src/objects/database.lzz

```diff
@@ -122,6 +140,126 @@ public:
 	}
 
 private:
+	class UpdateHook {
+	public:
+
+		explicit UpdateHook(
+			Addon * addon,
+			v8::Isolate * isolate,
+			Database * db,
+			v8::Local<v8::Function> fn
+		) :
+			addon(addon),
+			isolate(isolate),
+			db(db),
+			fn(isolate, fn) {}
+
+		static void invoke(void * data, int op, char const * dbName, char const * tableName, sqlite3_int64 rowid) {
```

Comment:
Since these hooks don't work with `WITHOUT ROWID` tables or with the [truncate optimization](https://www.sqlite.org/lang_delete.html#truncateopt), I think that should be called out in the docs.